certbot/display/util.py

```diff
@@ -164,6 +178,7 @@ def yesno(self, message, yes_label="Yes", no_label="No", **unused_kwargs):
         :rtype: bool
 
         """
+        assert OK == self.dialog.DIALOG_OK, "What kind of absurdity is this?"
```

Comment:
nit: If we want to leave this in, maybe we should check it at the top of the file or in `__init__` to catch the problem if we actually think there might be one early.